clippy.toml

```diff
@@ -0,0 +1,2 @@
+cyclomatic-complexity-threshold = 30
+doc-valid-idents = ["MiB", "GiB", "TiB", "PiB", "EiB", "SQLite", "PostgreSQL"]
```

Comment:
Those are clippy's default values, copied from the docs. There are actually defined [here](https://github.com/Manishearth/rust-clippy/blob/c66e90303fe7cd0df6f3191d157e65f4d58b5de5/src/utils/conf.rs#L148-L161) and I just noticed they also included 'GitHub'. I guess we could just leave them and add diesel's own CamelCase-names-that-are-not-code when we see an invalid warning?